### PR TITLE
Align Logo

### PR DESCRIPTION
**Pull request by janrenz**
screenshot 2018-11-07 at 03 02 40

this was annoying me for a while now

### CODE REVIEW

**Comment by schurigh** [comment]
Dont know if thats a bit too small for my liking. Maybe width: 40px; margin-right: 4px; ?
@adrianjost

**Comment by janrenz** [comment]
40 / -4 would be fine for me as well as long as the baseline is align.. @adrianjost Feel free to adapt this PR

**Comment by tofixx** [comment]
looks perfectly aligned

File: static/styles/lib/loggedin.scss [hunk 1]
```diff
@@ -108,19 +108,21 @@ aside.nav-sidebar {
...
             background-size: contain;
             @include media-breakpoint-down(md) {
                 width: 100%;
                 margin-right: 0;
+                margin-left: 0px;
             }
             @include media-breakpoint-down(xs) {
                 width: 45px;
                 margin-right: 0;
+                margin-left: 0px;
```
Comment (adrianjost) [inline static/styles/lib/loggedin.scss]: Isn't that line duplicated code?

**Pull request closed without merging**